### PR TITLE
cleaup unused files and parameterize cds_ip in presclient.

### PR DESCRIPTION
**Pull request by troy2914**

### CODE REVIEW

File: roles/p100/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+cds_ip: 10.3.3.207
```
Comment (vmcj) [inline roles/p100/defaults/main.yml:2]: if we put this as a var in the `./vars` we can use it here without needing to overwrite it everywhere.

File: roles/p100/templates/pres_client_run.sh.j2 [hunk 2]
```diff
@@ -19,7 +19,7 @@ gsettings set org.gnome.desktop.session idle-delay 0
 
 sleep 5
 while true; do
-  java -Xmx1024m -cp "lib/*" org.icpc.tools.presentation.contest.internal.ClientLauncher https://10.3.3.207:9443/api/contests presentation {{ cds_p100_password }} --name team
+  java -Xmx1024m -cp "lib/*" org.icpc.tools.presentation.contest.internal.ClientLauncher https://{{ cds_ip }}:9443/api/contests presentation {{ cds_p100_password }} --name team
```
Comment (vmcj) [inline roles/p100/templates/pres_client_run.sh.j2:22]: I would directly also parameterize the `9443` to make reuse easier for other contests.

File: roles/p100/defaults/main.yml (hunk 1 above)
Comment (troy2914) [reply roles/p100/defaults/main.yml:2]: wouldn't vars override the defaults ?

File: roles/p100/templates/pres_client_run.sh.j2 (hunk 2 above)
Comment (troy2914) [reply roles/p100/templates/pres_client_run.sh.j2]: done, also asking Tim about parameterizing the endpoint.

File: roles/p100/defaults/main.yml (hunk 1 above)
Comment (troy2914) [reply roles/p100/defaults/main.yml:3]: i'm thinking if we wind up using 2 cds's,  either wf46 or wf47 would get a vars for cds_ip to change it's ip.
and per https://docs.ansible.com/ansible/2.9/user_guide/playbooks_variables.html#playbooks-variables  that is the preferred way to do it.

**Review by vmcj — APPROVED** [review]

**Pull request merged**